compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -171,7 +171,18 @@ void TR_X86ProcessorInfo::initialize(bool force)
             uint32_t extended_model = getCPUModel(_processorSignature) + (getCPUExtendedModel(_processorSignature) << 4);
             switch (extended_model)
                {
-               case 0x55:
+               case 0xcf:
+                  _processorDescription |= TR_ProcessorIntelEmeraldRapids; break;
+               case 0x8f:
+                  _processorDescription |= TR_ProcessorIntelSapphireRapids; break;
+               case 0x6a:  // IceLake_X
+               case 0x6c:  // IceLake_D
+               case 0x7d:  // IceLake
+               case 0x7e:  // IceLake_L
+                  _processorDescription |= TR_ProcessorIntelIceLake; break;
+               case 0x55:  // Skylake_X
+               case 0x4e:  // Skylake_L
+               case 0x5e:  // Skylake
```

Comment:
Whats the meaning behind the suffix in your comment?

Reply:
I added a comment in my latest forced push.  I borrowed the suffix conventions for the processors we recognize from the Linux kernel.